frontend/src/components/story/StoryDay.vue

```diff
@@ -1,54 +1,59 @@
 <template>
-  <div class="e-story-day contents">
-    <h3 class="body-2 grey--text text--darken-2 e-story-day-title">
-      {{ dateLong(day.start) }}
-    </h3>
-    <template v-if="loading">
-      <v-skeleton-loader class="mt-2 mt-sm-3" type="list-item-three-line" />
-    </template>
-    <template v-else-if="entriesWithStory.length">
-      <template v-for="{ scheduleEntry, storyChapters } in entriesWithStory">
-        <div v-for="chapter in storyChapters" :key="chapter._meta.uri">
-          <h4 class="mt-2 mt-sm-3">
-            <span class="d-inline-flex align-center">
-              <span class="tabular-nums">{{ scheduleEntry.number }}</span>
-              <CategoryChip :category="scheduleEntry.activity().category" class="mx-1" />
-            </span>
-            <router-link
-              :to="{
-                name: 'activity',
-                params: {
-                  campId: day.period().camp().id,
-                  scheduleEntryId: scheduleEntry.id,
-                },
-              }"
-              class="e-title-link"
-            >
-              <span>{{ scheduleEntry.activity().title }}</span>
-              <template v-if="chapter.instanceName">
-                - {{ chapter.instanceName }}
-              </template>
-            </router-link>
-          </h4>
-          <api-form :entity="chapter">
-            <api-richtext
-              :outlined="false"
-              :solo="false"
-              auto-grow
-              dense
-              :readonly="!editing"
-              fieldname="data.text"
-              aria-label="Erfassen"
-              label=""
-            />
-          </api-form>
-        </div>
+  <v-expansion-panel>
+    <v-expansion-panel-header>
+      <h3>{{ dateLong(day.start) }}</h3>
+    </v-expansion-panel-header>
+    <v-expansion-panel-content>
+      <template v-if="loading">
+        <v-skeleton-loader class="mt-2 mt-sm-3" type="list-item-three-line" />
```

Comment:
You could omit the `<template>` here and move the `v-if` to the `<v-skeleton-loader>`

Reply:
[8608020](https://github.com/ecamp/ecamp3/pull/3038/commits/86080202531df5b06be0648e5ebfc87aebdac007)